frontend/src/app/Settings/Facility/facilitySchema.ts

```diff
@@ -6,6 +6,9 @@ import {
   isValidCLIANumber,
   stateRequiresCLIANumberValidation,
 } from "../../utils/clia";
+// import {
```

Comment:
Did you leave these commented lines in here intentionally? If not, you may want to consider removing them, to keep the code clean.

Reply:
I did not, thanks for catching!